content/en/tutorials/lambda-ecr-container-images/index.md

```diff
@@ -0,0 +1,195 @@
+
+## Prerequisites
+
+For this tutorial you will need:
+
+- [LocalStack Pro](https://localstack.cloud/pricing/) to emulate Amazon ECS and AWS Lambda locally
+  - Don't worry, if you don't have a subscription yet, you can just get a trial license for free.
+- [awslocal](https://docs.localstack.cloud/integrations/aws-cli/#localstack-aws-cli-awslocal)
+- [Docker](https://docker.io/)
+
+## Creating a Lambda function
+
+To package & deploy a Lambda as a container image, we must need to create a Lambda function containing our code and a Dockerfile. Create a new directory and initialize create two files: `handler.py`, to save our Python-based Lambda function, and `Dockerfile`, to package our code and dependencies into a container image.
+
+{{< command >}}
+$ mkdir -p lambda-container-image
+$ cd lambda-container-image
+$ touch handler.py Dockerfile
+{{< / command >}}
+
+Let us use the following Python code to create a Lambda function that returns a simple `'Hello from LocalStack Lambda container image!'` message.
+
+```python
+def handler(event, context):
+    print('Hello from LocalStack Lambda container image!')
+```
+
+In the above example, the `handler` function is executed by the Lambda service every time a trigger event occurs. The above function serves as an entrypoint for the Lambda function inside a runtime environment and accepts `event` and `context`, to receive information about the event and the invocation properties.
+
+## Building the image
+
+To package our Lambda function as a container image, we need to create a Dockerfile that contains the instructions to build our image. The image should be able to execute a read-only file system with access to a `/tmp` directory. We would be using [AWS base images for Lambda](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-images.html#runtimes-images-lp) to preload the runtimes and dependencies that are required to create Lambda images.
+
+To build the container image for Python, we would use a `python:3.8 base image`. Open the `Dockerfile` and specify the function handler to ensure that the Lambda runtime can locate it where the Lambda handler is available:
+
+```Dockerfile
+FROM public.ecr.aws/lambda/python:3.8
+
+COPY ./handler.py ./
+
+CMD [ "handler.handler" ]
+```
+
+{{< alert title="Note" color="primary">}}
+If you have additional dependencies specificed, create a file named `requirements.txt` and list the libraries in the file. Install the dependencies in the Dockerfile under the `${LAMBDA_TASK_ROOT}` directory.
+{{< /alert >}}
+
+Use Docker to build an image using this function code:
+
+{{< command >}}
+$ docker build -t localstack-lambda-container-image .
+{{< / command >}}
+
+## Publishing the image to ECR
+
+Now that the initial setup is done, we can give LocalStack's AWS emulation a try to push our image to ECR and deploy the Lambda container image. Let's start LocalStack:
+
+{{< command >}}
+$ LOCALSTACK_API_KEY=<your-api-key> localstack start -d
```

Comment:
nit: not sure this is really required - I think we can generally assume that LocalStack is already started (or refer to the getting started section). Thoughts?

Reply:
I believe that we can still keep this section around, since the tutorial serve the information end-to-end and in this way, since we can expect people to try out these tutorials without even looking at rest of the documentation. What do you think?